### PR TITLE
Rahul ne ek naya rg banaya

### PR DESCRIPTION
**Pull request by mq3274**
manager bola

### CODE REVIEW

File: rg.tf [hunk 1]
```diff
@@ -0,0 +1,9 @@
+resource "azurerm_resource_group" "example" {
+  name     = "my-rg"
+  location = "West Europe"
+}
+
+resource "azurerm_resource_group" "example" {
+  name     = "rg-rahul"
+  location = "Central India"
```
Comment (Githbbuddy) [inline rg.tf:8]: location galat hai East US dalo

Comment (mq3274) [reply rg.tf]: done sir

Comment (Githbbuddy) [reply rg.tf]: ok good

**Review by Githbbuddy — APPROVED** [review]
all good

**Pull request merged**